### ISSUE TITLE
"Next Play" doesn't update in current.json

### ISSUE THREAD

**Issue by ChoiZ** [yomguy/DeeFuzzer#78]
In the metadata section the next play time doesn't update.

**Comment by yomguy** [comment]
That's right, the trigger has been removed in the last pull requests and will be re-added soon!
Thanks @ChoiZ for the warning.
